### DISCUSSION TITLE
Throttle full forward when applying reverse thrust

### DISCUSSION THREAD

**Discussion by Rubenzxl** [magknight/787aviatorsTracker#520] [Completed things] [answered]
**Describe the bug**
When applying reverse thrust the throttles go full forward. Vid: https://youtu.be/gxIJqFagjpY 

**To Reproduce**
Apply reverse thrust

**Expected behavior**
I expect the throttle to be in idle position and only the reverse thrust levers to go up

**Desktop (please complete the following information):**
 - OS: Windows 10
 - X-Plane version: 11.41
 - Version: 1.6.0 cygnus 5

**Comment by aeroniemi** [comment]
Is this different to pre-new cockpit behaviour?

**Comment by Rubenzxl** [comment]
just checked, it's the same way in the old cockpit. I never noticed it. But seems like a bug to me?

**Comment by aeroniemi** [comment]
it's more just the way that X-Plane handles them  and the way most people
use throttles, but I guess we could change it

**Comment by DJ-STRE** [comment]
@Rubenzxl  Make sure you dont have ''full reverse thrust'' enabled in the xplane joystick/yoke settings

**Comment by aeroniemi** [comment, marked as answer]
Behaviour was changed somewhere in the early v1.7.x run - not sure entirely where though
